src/sentry/static/sentry/app/views/eventsV2/eventView.tsx

```diff
@@ -865,6 +865,13 @@ class EventView {
     return eventQuery;
   }
 
+  getResultsViewUrlTarget(organization: OrganizationSummary): LocationDescriptor {
```

Comment:
We can start using this as there are a few places we do navigation based on an event view.